.devcontainer/Dockerfile

```diff
@@ -5,6 +5,17 @@ FROM ubuntu:20.04
 # Instructs apt-get to run without a terminal
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Terraform, providers and tflint versions
+ARG TERRAFORM_VERSION=1.0.0
+ARG AZURERM_VERSION=2.63.0
+ARG RANDOM_VERSION=3.1.0
+ARG TIME_VERSION=0.7.1
+ARG TFLINT_VERSION=0.29.1
+ARG TFLINT_AZURERM=0.10.1
+
+# Azure CLI version
+ARG AZURE_CLI_VERSION=2.25.0-1~focal
```

Comment:
Since we're running into compatibility issues we discussed offline with az CLI 2.25, to make this change clear and single-purpose, (rather than introduce behavioral changes in other scripts as a result of moving to 2.25) let's return this to 2.22:

```suggestion
# Azure CLI version
ARG AZURE_CLI_VERSION=2.22.0-1~focal
```